build.gradle

```diff
@@ -744,19 +744,42 @@ dependencies {
 	}
 }
 
-remapJar {
-	afterEvaluate {
-		subprojects.each {
-			if (it.name in devOnlyModules || metaProjects.contains(it.name)) {
-				return
-			}
+configurations {
+	nestedJars {
+		transitive = false
+	}
+}
 
-			// Include the signed or none signed jar from the sub project.
-			nestedJars.from project("${it.path}").tasks.getByName("signRemapJar")
+dependencies {
+	subprojects.each {
+		if (it.name in devOnlyModules || metaProjects.contains(it.name)) {
+			return
 		}
+
+		nestedJars project("${it.path}")
+	}
+}
+
+remapJar {
+	nestedJars.from configurations.nestedJars
+}
+
+// Attempt to create a single jar with all files from all nested jars, this will fail if there are duplicate files.
+tasks.register("checkNoDuplicateFiles", Zip) {
```

Comment:
This could be a `Copy` too, but I guess this saves some disk space since the files aren't even used

Reply:
Yes it could be a `Sync`, However I think Zip is likely best, if I had to guess zip is likely faster than dealing with 1000s of small files. In the end it doesnt really matter.